src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -133,7 +134,19 @@ export class SmartSnippet extends Component {
      *
      * Default value is `undefined`.
      */
-    hrefTemplate: ComponentOptions.buildStringOption()
+    hrefTemplate: ComponentOptions.buildStringOption(),
+
+    /**
+     * Specify if the SmartSnippet should be displayed inside an iframe or not.
+     *
+     * Use this option in specific cases where your environment has limitations around iframe usage.
+     *
+     * **Examples:**
+     *
+     * ```html
+     * <div class='CoveoSmartSnippet' data-without-frame='true'></div>
+     */
+    withoutFrame: ComponentOptions.buildBooleanOption({ defaultValue: false })
```

Comment:
I would prefer withIFrame defaulting to true rather than a negation, but it's not crucial

Reply:
I have taken that habit because in LWC at least, when options can be defined in HTML, simply having the attribute there can be counted as true

so something like `<div class="..." without-frame>` is the same as `<div class="..." without-frame="true">`

But that's not the case in the JSUI so I am open to changing it, I don't like negatives either.